<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationWindow</class>
 <widget class="QMainWindow" name="SimulationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Cannonball Simulator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/assets/mainWindowIcon.png</normaloff>:/assets/mainWindowIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="promptCurrentScoreLabel">
      <property name="text">
       <string>Current Score:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="currentScoreLabel">
      <property name="styleSheet">
       <string notr="true">color: red; font-size: 16pt;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QGroupBox" name="difficultyGroupBox">
      <property name="title">
       <string>Difficulty</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="1" column="0">
        <widget class="QRadioButton" name="levelOneRadioButton">
         <property name="text">
          <string>Level 1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="levelTwoRadioButton">
         <property name="text">
          <string>Level 2</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="levelThreeRadioButton">
         <property name="text">
          <string>Level 3</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="pauseRadioButton">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QTextBrowser" name="tutorialTextBrowser">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="promptInitialAngleLabel">
      <property name="text">
       <string>Initial Angle [Deg]:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="initialAngleDoubleSpinBox">
      <property name="maximum">
       <double>90.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="promptInitialVelocityLabel">
      <property name="text">
       <string>Initial Velocity [m/s]:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="initialVelocityDoubleSpinbox">
      <property name="maximum">
       <double>20.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="firePushButton">
      <property name="styleSheet">
       <string notr="true">background: #ff0000;</string>
      </property>
      <property name="text">
       <string>Fire!</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QPushButton" name="storePushbutton">
      <property name="styleSheet">
       <string notr="true">background: #ffdf00;</string>
      </property>
      <property name="text">
       <string>Store</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QPushButton" name="youtubePushButton">
      <property name="text">
       <string>YouTube</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/assets/youtubeIcon.png</normaloff>:/assets/youtubeIcon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
